vmdb/app/models/user.rb

```diff
@@ -915,6 +915,11 @@ def self.current_userid=(userid)
     Thread.current[:userid] = userid
   end
 
+  def self.current_user=(user)
+    Thread.current[:user]   = user
+    Thread.current[:userid] = user.try(:userid)
+  end
```

Comment:
o_O We already had thread local stuff here, so why the new method?  I would think we would just use the existing methods.

What is `user` variable in this context?  If it's an instance of user, are we sure we want to do that...it's essentially caching what wasn't cached previously, and may lead to issues if the record itself.  I'm not sure if any UI code is relying on that aspect.